build/images/ovs/build.sh

```diff
@@ -141,6 +140,11 @@ if [ "$DISTRO" == "ubuntu" ]; then
     docker_build_and_push "antrea/openvswitch" "Dockerfile"
 elif [ "$DISTRO" == "ubi" ]; then
     docker_build_and_push "antrea/openvswitch-ubi" "Dockerfile.ubi"
+elif [ "$DISTRO" == "windows" ]; then
+    image="antrea/windows-ovs"
+    cache_args=$(get_cache_args $image $BUILD_TAG $PUSH $NO_CACHE)
+    build_args="--build-arg OVS_VERSION=$OVS_VERSION"
+    docker_build_and_push_windows "${image}" "Dockerfile.windows" "${cache_args}" "${build_args}" "${OVS_VERSION}" $PUSH
```

Comment:
As we discussed previously, I don't think it makes sense to use registry-based build caching for Windows, so I'm not sure we should call `get_cache_args` at all here. And if you agree with this, then I don't think we should even introduce `get_cache_args` as it makes this PR larger than it needs to be.

Reply:
reverted the change.